ci/pipelines/deploy-to-staging.yml

```diff
@@ -1174,11 +1198,13 @@ jobs:
   - name: deploy-ledger-to-staging
     plan:
       - get: ledger-ecr-registry-staging
-        trigger: true
```

Comment:
How come we're removing the trigger here?

Reply:
Ah it shouldn't be in production either! We should only be triggering things into staging + prod which aren't in the CDE.